### DISCUSSION TITLE
General error: 1364 Field 'last_name' doesn't have a default value

### DISCUSSION THREAD

**Discussion by Ceven47** [orchidsoftware/platform#2213] [Q&A]
I'm having this issue and i don't seem to know how to resolve it.

SQLSTATE[HY000]: General error: 1364 Field 'last_name' doesn't have a default value (SQL: insert into `users` (`name`, `email`, `password`, `permissions`, `updated_at`, `created_at`) values (admin, admin@admin.com, $2y$10$FksRUlgrF2z4VQ0EfI2TPO3eaqKiGjItxy/FCC02FWZcwHf1JcT6a, {"platform.systems.roles":true,"platform.systems.users":true,"platform.systems.attachment":true,"platform.index":true}, 2022-04-26 10:33:27, 2022-04-26 10:33:27))

**Comment by c-gross** [comment]
last_name is not part of the orchid user installation. Thats something you or an other package added.
